Classes/Repository/RepositoryViewController.swift

```diff
@@ -142,21 +142,16 @@ ContextMenuDelegate {
     }
 
     func newIssueAction() -> UIAlertAction? {
-        guard let newIssueViewController = NewIssueTableViewController.create(
-            client: client,
-            owner: repo.owner,
-            repo: repo.name,
-            signature: .sentWithGitHawk)
-        else {
-            Squawk.showGenericError()
-            return nil
+        let action = UIAlertAction(title: "New Issue", style: UIAlertActionStyle.default) { _ in
```

Comment:
`"New Issue"` should be localized.

Reply:
I changed this in both createWithTemplate(..) and create(..), I'm going to move all the localized strings to the Constants file as well